src/ifd/ifd.rs

```diff
@@ -159,6 +159,20 @@ impl Ifd {
         self
     }
 
+    /// Returns the same `Ifd`, after adding the specified pairs of Tags and Values.
+    ///
+    /// Because it returns `Self`, it is possible to chain this method.
+    ///
```

Comment:
Could you add a # Panics section telling that "If the inserted entries already exist, this function will `panic`."?